src/pages/Filtering.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Layout from "../components/Layout";
+import { useQuery } from "react-query";
+
+function Filtering() {
+  const { isLoading, error, data } = useQuery("menu_items", () =>
+    fetch("https://demo5940257.mockable.io/menu_items").then((response) =>
+      response.json()
+    )
+  );
```

Comment:
This is a great spot for some `console.log` calls.  My guess is that nothing is coming back because calling `.json()` on the response is _also_ asynchronous, but you're not _awaiting_ the result.  Here's how I would approach troubleshooting this:

```suggestion
  const { isLoading, error, data } = useQuery("menu_items", () => {
    console.log('Fetch call started');
    return fetch("https://demo5940257.mockable.io/menu_items").then(async (response) => {
      console.log('fetch call happened; server responded')
      const result = await response.json()
      // If the line below is never called, it means the .json() call is failing because the server isn't 
         returning JSON; likely because of a non-200 response.
      console.log({result})
      return result
    )
  );
```